src/System.Private.CoreLib/shared/System/Threading/Tasks/ValueTask.cs

```diff
@@ -776,7 +777,7 @@ public TResult Result
             new ConfiguredValueTaskAwaitable<TResult>(new ValueTask<TResult>(_obj, _result, _token, continueOnCapturedContext));
 
         /// <summary>Gets a string-representation of this <see cref="ValueTask{TResult}"/>.</summary>
-        public override string ToString()
+        public override string? ToString()
```

Comment:
Is the return type nullable because we don't know what `TResult` is going to be?

Reply:
Yes.